qml/FileListPage.qml

```diff
@@ -241,6 +241,13 @@ Page {
                             listItem.deleteFile()
                         }
                     }
+                    MenuItem {
+                        //: Share a file
+                        //% "Share"
+                        text: qsTrId("sailfish-office-me-share")
+                        onClicked: pageStack.push("Sailfish.TransferEngine.SharePage",
+                                                   { serviceFilter: ["e-mail"], mimeType: model.fileMimeType, source: model.filePath })
+                    }
```

Comment:
Use the same order as in Settings->Storage so "Share" first and "Delete" 2nd context menu item.

Reply:
Updated